atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/Containers.java

```diff
@@ -104,7 +105,7 @@ private void setupLogCollectorForLogDirectory() throws InterruptedException {
         if (currentLogCollector != null) {
             currentLogCollector.stopCollecting();
         }
-        currentLogCollector = InterruptibleFileLogCollector.fromPath(logDirectory);
```

Comment:
nit: restore this factory please

Reply:
Rebased properly, is no longer in this PR.